package/Caddyfile

```diff
@@ -0,0 +1,53 @@
+﻿# The Caddyfile is an easy way to configure your Caddy web server.
```

Comment:
That would be nice to use the same Caddyfile in every distributions, with the same features enabled.

Reply:
I don't think it makes sense to enable a worker by default in the docker Caddyfile. There's no example project (and it makes no sense to ship one either).

Reply:
I've removed the worker mode from the default Caddyfile. However, I still don't think it really makes sense to use the same Caddyfiles for docker and system packages. They just inherently incur different conventions.